keystore/src/entities/platform/generic/mls/group.rs

```diff
@@ -14,10 +14,11 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see http://www.gnu.org/licenses/.
 
```

Comment:
This commit references `mls_group` table in its message, but the table is actually called `mls_groups`.